service/README.md

```diff
@@ -0,0 +1,56 @@
+# fpl-ccd-configuration
+
+[![Build Status](https://travis-ci.org/hmcts/spring-boot-template.svg?branch=master)](https://travis-ci.org/hmcts/spring-boot-template)
+ ## Building and deploying the application
+ ### Building the application
+ The project uses [Gradle](https://gradle.org) as a build tool. It already contains
+`./gradlew` wrapper script, so there's no need to install gradle.
+ To build the project execute the following command:
+ ```bash
+  ./gradlew build
+```
+ ### Running the application
+ Create the image of the application by executing the following command:
+ ```bash
+  ./gradlew assemble
+```
+ Create docker image:
+ ```bash
+  docker-compose build
+```
+ Run the distribution (created in `build/install/spring-boot-template` directory)
```

Comment:
Replace all references to `spring-boot-template` with something more appropriate.